test_conformance/images/clFillImage/test_fill_generic.cpp

```diff
@@ -117,7 +108,8 @@ cl_mem create_image( cl_context context, cl_command_queue queue, BufferOwningPtr
                 cl_mem_flags buffer_flags = CL_MEM_READ_WRITE;
                 if (gEnablePitch)
                 {
-                    if (CL_VERSION_MAJOR(version) == 1)
+                    int major = (version.to_int()) / 10;
```

Comment:
We should probably add `Version::major()` and `Version::minor()` getters to the harness code so users don't have to do math.